package.json

```diff
@@ -116,6 +116,7 @@
     "webpack-dev-server": "2.1.0-beta.9"
   },
   "scripts": {
+    "postinstall": "make install-selenium",
```

Comment:
But ... This will install selenium in the production server, so I'm not in favor of this.